packages/orbit-design-tokens/src/dictionary/definitions/component/heading/heading-line-height.json

```diff
@@ -0,0 +1,118 @@
+{
+  "component": {
+    "heading": {
+      "display": {
+        "line-height": {
+          "type": "size",
+          "value": 44
+        }
+      },
+      "display-subtitle": {
+        "line-height": {
+          "type": "size",
+          "value": 28
+        }
+      },
+      "title-1": {
+        "line-height": {
+          "type": "size",
+          "value": 36
+        }
+      },
+      "title-2": {
+        "line-height": {
+          "type": "size",
+          "value": 28
+        }
+      },
+      "title-3": {
+        "line-height": {
+          "type": "size",
+          "value": 24
+        }
+      },
+      "title-4": {
+        "line-height": {
+          "type": "size",
+          "value": 20
+        }
+      },
+      "title-5": {
+        "line-height": {
+          "type": "size",
+          "value": 20
+        }
+      },
+      "title-6": {
+        "line-height": {
+          "type": "size",
+          "value": 20
```

Comment:
This used to be 16

Reply:
I can change both on the Heading branch, but something is off here. I didn't manually check each value, but now we don't know if there are more values off

Reply:
You can check it in figma library in typography Orbit section. Fixed that, good catch

Reply:
Yeah, there are more inconsistent ones, according to Figma:
- title 3 should be 18px, not 16px (it is using foundation.fontSize.large but it should use XLarge)
- title 4 should be 16px, not 15px (it is using foundation.fontSize.normal but it should use Large)
- title 4 should have fontWeight 700, not 500 (it is using foundation.fontWeight.medium but it should use Bold) - this is actually wrong in the current Orbit implementation as well
- title 5 should be 15px, not 13px (it is using foundation.fontSize.small but it should use Normal)

Also we're missing the line height for xlarge for text, no? Something like `lineHeightTextLarge` with the value of 24px 💭

Reply:
Fixed that for headings. Line-height large is 24px, we do not have text extra large, or wdym?

Reply:
We do have text extra large, it was added last month. What happens is that the line height is the same on both text large and extra large, and so in React we are reusing the same token (`lineHeightTextLarge`). But shouldn't we have a token for each size, in this case, with the same value?

Reply:
I guess it make sense if in future the value might change